h/features/models.py

```diff
@@ -35,6 +35,23 @@
 #
 FEATURES_PENDING_REMOVAL = {}
 
+FEATURE_FEATURECOHORT_TABLE = sa.Table(
+    'feature_featurecohort', Base.metadata,
+    sa.Column('id',
+              sa.Integer(),
+              nullable=False),
+    sa.Column('feature_id',
+              sa.Integer(),
+              nullable=False),
+    sa.Column('cohort_id',
+              sa.Integer(),
+              nullable=False),
+    sa.PrimaryKeyConstraint('id'),
```

Comment:
Perhaps use the `primary_key` and `autoincrement` keyword arguments as you've done for the user join table.